lib/manageiq/appliance_console/cli.rb

```diff
@@ -351,6 +361,11 @@ def uninstall_ipa
       config.deactivate if config.ipa_client_configured?
     end
 
+    def scap
```

Comment:
Should the be `def openscap`?

Reply:
Whoops, yep, must've forgot to change this when I was copying it over from my test appliance.